financialaid/migrations/0003_added_timestamps.py

```diff
@@ -4,7 +4,7 @@
 
 import datetime
 from django.db import migrations, models
-
+import pytz
```

Comment:
It wouldn't help because the warning would occur on the first migration, even if the time zone is added on the second migration